### PR TITLE
Feature/ikm 20240919 hybrid reasoner

### PR DESCRIPTION
**Pull request by emays**

### CODE REVIEW

**Review by carldea — CHANGES_REQUESTED** [review]
@emays 
I have some comments and questions in the code review.

**Comment by emays** [comment]
The tests and especially the ITs are still a work in progress pending resolution of data issues. So some of that code that's commented out will be reviewed and potentially reinstated when the data issues are resolved.

**Review by stenkarl — APPROVED** [review]

File: reasoner/reasoner-elk-snomed/src/test/java/dev/ikm/tinkar/reasoner/elksnomed/SnomedUS20230901ElkSnomedClassifierTestIT.java [hunk 1]
```diff
@@ -71,7 +70,7 @@ public void isas() throws Exception {
 		LOG.info("Create reasoner");
 		SnomedOntologyReasoner reasoner = SnomedOntologyReasoner.create(ontology);
 
-		ViewCalculator viewCalculator = getViewCalculator();
+//		ViewCalculator viewCalculator = getViewCalculator();
 //		{
 ////			LOG.info("Props>>>>>");
 //			OWLOntology oo = ontology.getOntology();
```
[the remark this answers is not in the document]
Comment (rbradley813) [reply reasoner/reasoner-elk-snomed/src/test/java/dev/ikm/tinkar/reasoner/elksnomed/SnomedUS20230901ElkSnomedClassifierTestIT.java:76]: @emays said he will refactor the tests once the new data is provided

**Review by carldea — APPROVED** [review]
looks good to me.

**Pull request merged**